### DISCUSSION TITLE
Outbound Links - what is the definition?

### DISCUSSION THREAD

**Discussion by thinkplaymake** [plausible/analytics#1359] [Anything Else]
Hello

I'm looking to setup outbound link tracking - I'm assuming the javascript intelligently looks at the href attribute and tags anything which isn't on the same domain as an "outbound" link and tracks it as such, but what about subdomains, are they treated as outbound?

i.e. is clicking to content.domain.com from www.domain.com treated as outbound?

Can you share a little of the internals to understand how outbound links are defined?

thanks!

**Comment by ukutaht** [comment]
Hey!

You can check the source code to see what it does exactly.

https://github.com/plausible/analytics/blob/master/tracker/src/plausible.js#L101

If `location.host` and `link.host` are different, the link will be tracked. This means that a link to a subdomain will be treated as an outbound link and the click will be tracked.

**Reply by thinkplaymake** [reply]
awesome!
